File: app/check_desc_format.py
import json, re, pathlib

root = pathlib.Path('D:/风不死/编码/Code/Mimo/MBTI-Character-Test/app')
with open(root / 'src/data/archetypes.js', 'r', encoding='utf-8') as f:
    content = f.read()

# 提取 D1 high 的 desc
match = re.search(r"D1.*?desc:\s*`(.*?)`", content, re.DOTALL)
if match:
    desc = match.group(1)
    lines = desc.strip().split('\n')
    for i, line in enumerate(lines[:3]):
        line = line.strip()
        print(f"Line {i}: [{line[:20]}]")
        print(f"  First 5 chars: {[ord(c) for c in line[:5]]}")

File: app/check_desc_format.mjs
import { SPECIAL_HIGH } from '../src/data/archetypes.js'
const desc = SPECIAL_HIGH.D1.desc
const lines = desc.trim().split('\n').map(line => line.trim())
for (let i = 0; i < 3; i++) {
  const line = lines[i] || ''
  const first10 = line.substring(0, 10)
  const codes = [...first10].map(c => c.charCodeAt(0))
  console.log(`Line ${i}: "${first10}" -> ${codes.join(',')}`)
}
